### PR TITLE
:globe_with_meridians: Missing Dutch translations

### PR DESCRIPTION
**Pull request by jbostoen**
As discussed, just a few missing Dutch strings (+one change to be more consistent)

@Hipska this was the PR @piRGoif mentioned you for.

### CODE REVIEW

**Review by piRGoif — APPROVED** [review]

**Comment by piRGoif** [comment]
All good for me.
Waiting for Hipska feedback :)

File: dictionaries/nl.dictionary.itop.core.php [hunk 1]
```diff
@@ -210,12 +210,12 @@
...
-	'Core:Context=Synchro' => 'Synchro~~',
-	'Core:Context=Setup' => 'Setup~~',
-	'Core:Context=GUI:Console' => 'Console~~',
-	'Core:Context=CRON' => 'CRON~~',
-	'Core:Context=GUI:Portal' => 'Portal~~',
+	'Core:Context=REST/JSON' => 'REST',
+	'Core:Context=Synchro' => 'Synchro',
+	'Core:Context=Setup' => 'Setup',
+	'Core:Context=GUI:Console' => 'Console',
+	'Core:Context=CRON' => 'CRON',
```
Comment (Hipska) [inline dictionaries/nl.dictionary.itop.core.php]: `cron` is not an abbreviation, so should be in small letters. Pick either "Cron" or "cron". Applies to other languages as well.

**Review by Hipska — CHANGES_REQUESTED** [review]

File: dictionaries/nl.dictionary.itop.core.php (hunk 1 above)
Comment (piRGoif) [reply dictionaries/nl.dictionary.itop.core.php]: Indeed, I agree !
But all languages use CRON in capital letters for now... 
I added a ticket to change this in 2.7.1 (N°2866). I'd prefer not to change this for 2.7.0 => I'm resolving this conversation.

**Comment by piRGoif** [comment]
Merging this PR as there is just one feedback, but that will be processed later.

File: dictionaries/nl.dictionary.itop.core.php (hunk 1 above)
Comment (Hipska) [reply dictionaries/nl.dictionary.itop.core.php]: Okay, perfect.

**Pull request merged**